install_config/upgrading/manual_upgrades.adoc

```diff
@@ -390,6 +390,35 @@ upgrade will fail and should be restarted automatically. This will not disrupt
 pods that are already running.
 ====
 
+[[enforcing-quota-in-the-registry]]
+=== Enforcing quota in the Registry
+
+To prevent layer blobs exceeding size limit from being written to registry's
+storage, quota must be enforced. This can be achieved either via a
+link:../install/docker_registry.html#registry-configuration-reference[configuration
+file]:
+
+====
+----
+...
+middleware:
+  repository:
+    - name: openshift
+      options:
+        enforcequota: true
+...
+----
+====
+
+Or with an environment variable
+`*REGISTRY_MIDDLEWARE_REPOSITORY_OPENSHIFT_ENFORCEQUOTA*`. The environment
+variable is set to `*true*` for the new registry deployments by default.
+Existing deployments need to be modified using:
+
+----
+# oc env dc/docker-registry REGISTRY_MIDDLEWARE_REPOSITORY_OPENSHIFT_ENFORCEQUOTA=true
+----
```

Comment:
@miminar Same here as in https://github.com/openshift/openshift-docs/pull/2634#pullrequestreview-12565392: Can I remove this section from the manual upgrade docs for 3.3 -> 3.4 since it would have already been done in 3.3?